scripts/messages/message_template.msg.sha1

```diff
@@ -1 +1 @@
-d7915d67467e59287857630bd89bf9529d065198
+d7915d67467e59287857630bd89bf9529d065199
```

Comment:
Why did the message template change?

Reply:
I think this one gets pulled automatically from server in some cases by TemplateCheck.cmake.

Reply:
It was changed automatically during the compilation. Removed from PR, thanks